addon/components/universal-export/file/component.js

```diff
@@ -71,19 +71,19 @@ export default Component.extend({
     'exportLimit.@each.spent',
     'selectedCount',
     function() {
-      let limitPath = `exportLimit.${this.get('selectedType')}.limit`;
-      let spentPath = `exportLimit.${this.get('selectedType')}.spent`;
+      let limitPath = `exportLimit.${this.selectedType}.limit`;
+      let spentPath = `exportLimit.${this.selectedType}.spent`;
 
       if (this.get(limitPath) === -1) {
```

Comment:
```suggestion
      if (limitPath === -1) {
```
Possibility to update with something like this?

Reply:
same as the comment for spentPath